de.seronet_projekt.xtend.ROS.generator/src/de/seronet_projekt/xtend/ROS/generator/ROS_Callbacks.xtend

```diff
@@ -132,15 +132,15 @@ class ROS_Callbacks {
 	«FOR sub: comp.allROSPorts.filter[it.port instanceof RosSubscriber]»
 		void «comp.name»RosPortCallbacks::«sub.name»_cb (const «sub.packageString»::«sub.messageString»::ConstPtr &msg)
 		{
-			//FIXME: implement this upcall method, use "COMP->" to access the component's class members
+			// for implementing this method, you can use the "COMP->" macro to access the component's class members
```

Comment:
can this be part of a separate commit/PR to be merged together with an update of the documentation https://wiki.servicerobotik-ulm.de/_detail/tutorials:ros:rosgeneratedcallback.png?id=tutorials%3Aros%3Amixed-port-component-ros  ??

Reply:
really? I don't think that a simple comment update justifies the effort of creating a new PR, this is just wasted time...

Reply:
>really? I don't think that a simple comment update justifies the effort of creating a new PR, this is just wasted time...

It is not a waste of time.
In general independent changes should be contained in separate commits.